soc/xtensa/intel_adsp/ace/Kconfig.defconfig.ace15_mtpm

```diff
@@ -11,4 +11,10 @@ config SOC
 config MP_MAX_NUM_CPUS
 	default 3
 
+config CRYPTO_INTEL_SHA
+	default y if DT_HAS_INTEL_ADSP_SHA_ENABLED
```

Comment:
We should move this into drivers/crypto/Kconfig.intel:

```
config CRYPTO_INTEL_SHA
        bool "Intel SHA driver"
        default y
        depends on DT_HAS_INTEL_ADSP_SHA_ENABLED
```

Reply:
The string is really needed ? It is already set somewhere else. It is like `MPMAX_NUM_CPUS`

Reply:
what do you mean?  which string?

Reply:
` bool "Intel SHA driver"`

Reply:
Take a look in others options in this file, none of them have this "help" text because they are defined somewhere else.

Reply:
I'm confused.  All I'm saying is instead of doing the `default y if DT_HAS_INTEL_ADSP_SHA_ENABLED` in Kconfig.defconfig.ace15_mtpm.  Move this logic into drivers/crypto/Kconfig.intel.

Reply:
ah ok, misunderstood it